spring-boot/admin/src/main/resources/templates/admin/fragments/scripts.html

```diff
@@ -23,6 +23,57 @@
             <script src="/webjars/github-com-jaxkodex-jQuery-Smart-Wizard/3.3.1/js/jquery.smartWizard.js"></script>
         </div>
 
+        <div th:fragment="apiKeyRemoveDialog">
+        <script>
+            $(function () {
+                $("#infoDialog").dialog({
+                    width: 'auto',
+                    height: "auto",
+                    autoOpen: false,
+                    modal: true,
+                    show: {
+                        effect: "blind",
+                        duration: 1500
```

Comment:
Assuming that this is the time it takes the dialog to "open", it should be shortened to e.g. 500

Reply:
I found 1000 to be better looking. A value of 500 looks "rushed" imo.